static/js/core.js

```diff
@@ -50,7 +50,7 @@ $.extend({
                 return m[1].toUpperCase();
             });
         }
-        return (reg).test(MEDUSA.config[pyVar]);
+        return (reg).test(MEDUSA.config.layout[pyVar]);
```

Comment:
Ok I thought it was, as isMeta() was only used by the layouts. Then I suggest you fix it xo.